### DISCUSSION TITLE
I've been waiting for 5 days and still haven't gotten the Github student pack.

### DISCUSSION THREAD

**Discussion by ax-6** [community/community#131840] [GitHub Education] [answered]
### Hi everyone,

**I've been waiting for 5 days and still haven't gotten the Github student pack.**
Below are screenshots:
![education github com_discount_requests_application](https://github.com/community/community/assets/123625244/d868898f-e2ca-4317-aefe-151950635fce)

**Comment by ax-6** [comment]
In [https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits), it is stated that for more than 5 days please contact the Education Community Team. So I hope this will be resolved soon.

**Comment by ax-6** [comment, marked as answer]
I have received an email from Github Education. The issue has been resolved.

**Comment by Rashmi-002** [comment]
My request is Approved But almost 3days But i didn't get the Pack still it shows will soon what can i do Guys i need to Work with my Projects Anyone please Help me...
